repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql

```diff
@@ -0,0 +1,6 @@
+DROP INDEX idx_reference  CASCADE;
+DROP INDEX idx_event__case_type_id  CASCADE;
+DROP INDEX event_pre_state_state_id_event_id_idx  CASCADE;
+DROP INDEX idx_field_type__id__base_field_type_id__collection_field_type_i  CASCADE;
+DROP INDEX idx_jurisdiction__id__reference  CASCADE;
+DROP INDEX idx_field_type_list_item__field_type_id  CASCADE;
```

Comment:
I've run the same SQL to identify duplicate indexes locally and see 8, but there's only 6 in here. Were the others intentionally excluded?